owldiff-gui/README.md

```diff
@@ -0,0 +1,54 @@
+<p align="center">
```

Comment:
This looks more like a generic Gatsby README file - would expect instructions how to work with OwlDiff GUI